include/bluetooth/att.h

```diff
@@ -47,6 +47,10 @@ extern "C" {
 /* Version 5.2, Vol 3, Part F, 3.2.9 defines maximum attribute length to 512 */
 #define BT_ATT_MAX_ATTRIBUTE_LEN		512
 
+/* Handle 0x0000 is reserved for future use */
+#define BT_ATT_FIRST_ATTTRIBUTE_HANDLE		0x0001
```

Comment:
Just FYI, the indentation is right, this is just Github that is bad a showing tabs properly :)

Reply:
Should use spaces for alignment though.

Reply:
Doesn't the style guide say to use whatever the file currently uses? 

If switching to spaces, it should be done for all the #defines, and is out of scope of this PR imo.

Reply:
Does it? I'm not sure, but I use spaces everywhere. I think most files has a mix of both at this point.
No big deal though.

Reply:
Hmm, looks like it's only for certain things that you should adopt what's already used in the code. Updated to use spaces.